<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>253</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Voreinstellungen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Voreinstellungen auswählen:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Zentralkörper</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Zweitkörper</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start-Distanz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="choice_central">
        <item>
         <property name="text">
          <string notr="true">Sun</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Earth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Moon</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="choice_sat">
        <item>
         <property name="text">
          <string notr="true">Earth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Moon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Sputnik2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="choice_distance">
        <item>
         <property name="text">
          <string notr="true">AU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">EarthMoon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">EarthSat</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="b_cancel">
        <property name="text">
         <string>Abbrechen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_fill">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Werte eintragen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Übernehmen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ok">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Übernehmen und schliessen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptionen">
    <property name="title">
     <string>Optionen</string>
    </property>
    <addaction name="actionVerlassen"/>
   </widget>
   <addaction name="menuOptionen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVerlassen">
   <property name="text">
    <string>Verlassen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>b_cancel</tabstop>
  <tabstop>b_fill</tabstop>
  <tabstop>b_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
